src/components/molecules/InformationRow/InformationRow.jsx

```diff
@@ -0,0 +1,25 @@
+import PropTypes from 'prop-types';
+import { Text } from '../../atoms/Text/Text';
+import { Icon } from '../../atoms/IconComponent/Icon';
+import './InformationRow.scss';
+
+export const InformationRow = ({ onClick, children, src }) => {
+    return (
+        <button onClick={onClick} className='information__row'>
```

Comment:
If this has to be a button, use our `Button` component. If not, follow @T0ffik's review.